source/Octopus.Cli/Commands/Releases/UnblockReleaseCommand.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Threading.Tasks;
+using Octopus.Cli.Infrastructure;
+using Octopus.Cli.Repositories;
+using Octopus.Cli.Util;
+using Octopus.Client;
+using Octopus.Client.Exceptions;
+using Octopus.Client.Model;
+
+namespace Octopus.Cli.Commands.Releases
+{
+    [Command("unblock-release", "resume-release-progression", Description = "Unblock a release from progressing to next phase.")]
```

Comment:
Oh, I missed the Alias on this command. I think `unblock-release` should be dropped and the description updated to match what it actually does